File: examples/php/organisation/update_organisation.php
<?php

$request = new HttpRequest();
$request->setUrl('https://exenzo.nl/api/v1/organisation/{organisation_id}');
$request->setMethod(HTTP_METH_PUT);

$request->setHeaders(array(
    'cache-control' => 'no-cache',
    'content-type' => 'application/json',
    'x-authorization' => 'ORGANISATION_API_TOKEN'
));

$request->setBody('{
    "name" : "Test Organisation Ltd"
}');

try {
    $response = $request->send();

    echo $response->getBody();
} catch (HttpException $ex) {
    echo $ex;
}

File: examples/python/organisation/update_organisation.py
import http.client

conn = http.client.HTTPSConnection("exenzo.nl")

payload = "{\n    \"name\" : \"Organisation Name\"\n}"

headers = {
    'x-authorization': "ORGANISATION_API_TOKEN",
    'content-type': "application/json",
    'cache-control': "no-cache",
    }

conn.request("PUT", "/api/v1/organisation/{organisation_id}", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))